doc/intro.md

```diff
@@ -58,38 +60,45 @@ where:
 
 ### Trend-based Cohort Component Model
 
-This method involves projecting births, deaths and migrations by age and sex.
+This method involves projecting births, deaths and migrations by age and sex and using that information to infer an overall population projection.
 
 See below the cohort component method we aim to provide:
```

Comment:
change to 'This flowchart shows the steps of the Cohort Component Model that occur within the core module and draw on the fertility, mortality and migration modules':